### PR TITLE
CONTRIBUTING.md update patch

### PR DESCRIPTION
**Pull request by Arsenic-ATG**
changes include :- 
1) updation of steps 2 and 3 in chapter 0 of the file
2) made **example** in **chapter 1/step 1** a drop down list ( to decrease the length of the entire file without effecting the amount of information )
3) add link to the LICENSE file under the `License` section fo the file

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,19 +8,20 @@ Ensure that your code and project is in a working condition and performs its int
 
 In order to ensure that your project can be accepted into our repo, please ensure your project has the
 following directories/folders and files.
-- **source/** -> This directory must contain the complete source code of your project
-- **output/** -> This directory must contain screenshots/videos of the working of your project, the
+- **source/** -> This directory must contain your acctual project including the complete source code of your project and the `.arproj` file.
```
Comment (sayak-sarkar) [inline docs/CONTRIBUTING.md]:
```suggestion
- **source/** -> This directory must contain your actual project including the complete source code of your project and the `.arproj` file.
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -8,19 +8,20 @@ Ensure that your code and project is in a working condition and performs its int
...
+- **output/** -> This directory must contain screenshots/videos/gifs of the working of your project, the
 purpose of this is to act as evidence that your project is in a working condition.
-- **README.md** -> The readme file will contain general information about your project. Please refer to Step 3 for the template of this readme file.
+- **README.md** -> The readme file will contain general information about your project. If possible also create a trial link to the project you have created on facebook or instagram in your readme, for further infomation please refer to Step 3 for the template of this readme file.
```
Comment (sayak-sarkar) [inline docs/CONTRIBUTING.md]:
```suggestion
- **README.md** -> The README file will contain general information about your project. If possible also create a trial link to the project you have created on Facebook or Instagram in your README, for further information please refer to Step 3 for the template of this README file.
```

**Review by sayak-sarkar — CHANGES_REQUESTED** [review]

**Comment by AJV009** [comment]
@Arsenic-ATG  dont you think there are tooo many commits? (6 commits to merge?)
Maybe you can squash them into one single commit.

**Comment by Arsenic-ATG** [comment]
@AJV009 branch managers can also do it at the time of merging, by selecting **squash and merge**. But if you want,  I can still squash them in one commit and open a new pull request with the same 🙂.

**Comment by AJV009** [comment]
Fine fine! 😅 Don't make another pr now! The manager will squash and commit them!

**Review by sayak-sarkar — APPROVED** [review]

**Pull request merged**